en/docs/Learn/DesignAPI/LifecycleManagement/api-lifecycle.md

```diff
@@ -1,14 +1,39 @@
 # API Lifecycle
 
-An API is the published interface, while the service is the implementation running in the backend. APIs have their own lifecycles that are independent to the backend services they rely on. This lifecycle is exposed in the API publisher web interface and is managed by the API publisher role.
+The API lifecycle is one of the key factors in API management. An API lifecycle has predefined states. These states represent the stages that an API has in the process of starting to develop an API until it's retirement. APIs provided by WSO2 API Manager has a lifecycle containing six stages which allows you to identify in which state that the API is currently on.
+
+##API lifecycle states
+
+The following stages are available in the default API lifecycle
+
+|   API Lifecycle State |   Use Case |   Action  |
+|-----------------------|------------|-----------|
+|   **CREATED**         |The API has been created, but it is not available for usage.|API metadata is added to the API Store, but it is not deployed in the API Gateway and therefore, is not visible to subscribers in the API Store|
```

Comment:
Shall we rename API store to developer portal?

Reply:
Fixed